content/en-us/reference/engine/classes/VoiceChatService.yaml

```
@@ -40,8 +38,23 @@ properties:
     capabilities: []
     writeCapabilities: []
   - name: VoiceChatService.UseAudioApi
-    summary: ''
-    description: ''
+    summary: |
+      Controls whether voice chat is represented and controlled by an `Class.AudioDeviceInput` object.
```

Since there's one per player
```suggestion
      Controls whether voice chat is represented and controlled by `Class.AudioDeviceInput` objects.
```